eyeball/Cargo.toml

```diff
@@ -2,14 +2,18 @@
 name = "eyeball"
 version = "0.7.0"
 edition = "2021"
+rust-version = "1.70.0"
 license = "MPL-2.0"
 description = "Add observability to your Rust types!"
 repository = "https://github.com/jplatte/eyeball"
 
 [dependencies]
 futures-core.workspace = true
 readlock.workspace = true
+readlock-tokio = { git = "https://github.com/jplatte/readlock", rev = "6c32222052fa75669c90cd418b06d115dd998d4e", optional = true }
```

Comment:
Why do we need to fork `readlock`?

Reply:
It's not a fork :P

But anyways, that git rev includes `SharedReadLock::lock_owned`. I'll fix up the docs and publish a release with it before merging this PR.

Reply:
Oh, you mean why there's a separate readlock-tokio crate? Because `readlock`'s types are based on `std::sync::RwLock`, and we need the same types, but based on `tokio`'s `RwLock`.

Reply:
Got it :-).